pages/linux/scrontab.md

```diff
@@ -0,0 +1,20 @@
+# scrontab
+
+> Manage Slurm crontab files.
+> More information: <https://slurm.schedmd.com/scrontab.html>.
+
+- Install a new crontab from the specified file:
```

Comment:
Should we place crontab inside backticks here or is it fine as it is?

Reply:
It is not used in the sense of the command `crontab`, but in the sense of the concept of a crontab. That's why i didn't put it in backticks.